lib/serverengine/supervisor.rb

```diff
@@ -44,6 +44,14 @@ def initialize(server_module, worker_module, load_config_proc={}, &block)
       @enable_detach = !!@config[:enable_detach]
       @exit_on_detach = !!@config[:exit_on_detach]
       @disable_reload = !!@config[:disable_reload]
+
+      @command_pipe = @config.fetch(:command_pipe, nil)
```

Comment:
It seems to be updated at L230 (like daemon.rb).
https://github.com/unak/serverengine/blob/7b485e72d58ac77f04132b4efef81e55fc22d33c/lib/serverengine/supervisor.rb#L230

Are there any way to use the configured value?